tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "module": "esnext",
     "moduleResolution": "node",
     "importHelpers": true,
-    "target": "es2015",
+    "target": "es2020",
```

Comment:
Why was this target change needed?

Reply:
It was changed automatically by the Angular CLI.